app/models/merchant.rb

```diff
@@ -1,3 +1,19 @@
 class Merchant < ApplicationRecord
-  
+  has_many :items
+  has_many :invoice_items, through: :items
+  has_many :invoices, through: :invoice_items
+  has_many :customers, through: :invoices
+  has_many :transactions, through: :invoices
+
+  validates :name, presence: true
+
+  def top_five_customers
```

Comment:
since we have this method in two different spots in our codebase I wonder if we could create a module for this method and mix it in to our admin and merchant controllers. 🤔

Reply:
Yes let's look into that!